bodhi/tests/test_style.py

```diff
@@ -47,9 +47,9 @@ def test_code_with_pydocstyle(self):
         The goal is that this test would one day check the entire codebase.
         """
         enforced_paths = [
-            'bodhi/server/scripts/__init__.py', 'bodhi/server/scripts/approve_testing.py',
-            'bodhi/server/scripts/clean_old_mashes.py', 'bodhi/server/scripts/expire_overrides.py',
-            'bodhi/server/scripts/initializedb.py']
+            'bodhi/server/consumers/__init__.py', 'bodhi/server/scripts/__init__.py',
```

Comment:
Nonblocking comment here:

I recommend breaking these out into one item per line.

It makes it much easier to review -- to see what was added and what was changed.